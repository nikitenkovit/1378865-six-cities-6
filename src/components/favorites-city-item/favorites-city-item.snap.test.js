import React from "react";
import {render} from "@testing-library/react";
import FavoritesCityItem from "./favorites-city-item";

const mockProps = {
  cityName: `Paris`,
  cityOffers: []
};

it(`Should FavoritesCityItem render correctly`, () => {
  const {container} = render(
      <FavoritesCityItem {...mockProps}/>);
  expect(container).toMatchSnapshot();
});
