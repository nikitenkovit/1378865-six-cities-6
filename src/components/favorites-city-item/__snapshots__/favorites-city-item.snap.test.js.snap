// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should FavoritesCityItem render correctly 1`] = `
<div>
  <li
    class="favorites__locations-items"
  >
    <div
      class="favorites__locations locations locations--current"
    >
      <div
        class="locations__item"
      >
        <a
          class="locations__item-link"
          href="#"
        >
          <span>
            Paris
          </span>
        </a>
      </div>
    </div>
    <div
      class="favorites__places"
    />
  </li>
</div>
`;
